src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -8711,7 +8790,23 @@ public void Inheritance_01()
 {
 }";
             var comp = CreateCompilation(source);
-            comp.VerifyDiagnostics();
+            comp.VerifyDiagnostics(
+                // (11,17): warning CS8907: Parameter 'P1' is unread. Did you forget to use it to initialize the property with that name?
+                // record B(object P1, object P2, object P3, object P4, object P5, object P6) : A
+                Diagnostic(ErrorCode.WRN_UnreadRecordParameter, "P1").WithArguments("P1").WithLocation(11, 17),
```

Comment:
It occurs to me that it's not really possible for the user to assign to the base property here. A scenario where we change a base nominal property to a positional parameter in a derived type is going to be unpleasant until we add the ability to include a constructor body for a primary constructor.